### DISCUSSION TITLE
buds recognition on kali linux

### DISCUSSION THREAD

**Discussion by sumerfat** [blueman-project/blueman#2269] [Q&A]
I have Linux kali 6.6.9 installed on mac air 2012 , but bluetooth doesn't recognize Xiaomi buds, it could find other devices like tv or phone   

* [x] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

**Comment by cschramm** [comment]
My immediate guess was that you're trying to find an Low-Energy-only device from a system that does not support LE. There's conflicting information on the Internets on whether a "MacBook Air (13-inch, Mid 2012)" has Bluetooth 4.0 hardware and if that does support LE. It seems like the first should generally be given, but there are counterexamples where it's either not 4.0 or at least does not support LE.
